scripts/playground/templates/airflow.yaml

```diff
@@ -50,6 +50,8 @@ spec:
             killed_task_cleanup_time: 604800
             dag_file_processor_timeout: 300
             dagbag_import_error_traceback_depth: 4
+            parallelism: 200
+            max_active_tasks_per_dag: 200
```

Comment:
I have increased to enable the concurrency when we go 100 HC.
the default value of `parallelism` is 32 per scheduler and `max_active_tasks_per_dag` is 16

Reply:
@mukrishn just for clarity, though, we may still be limited by the number of pods that airflow can spawn at one time. Since, to my understanding, each task is a new pod here.

Reply:
Yes, correct each task spawn a new pod. if the cluster reaches its limit the task will be `scheduled` but pod would be pending state till kubelet schedules it. We have 3 workers and at max we can do 3x220 pods approximately, we should also adjust our cron schedule accordingly to avoid over committing tasks.